operator/v1/types_ingress.go

```diff
@@ -1037,6 +1037,16 @@ type SyslogLoggingDestinationParameters struct {
 // ContainerLoggingDestinationParameters describes parameters for the Container
 // logging destination type.
 type ContainerLoggingDestinationParameters struct {
+	// maxLength is the maximum length of the syslog message
```

Comment:
Please use full sentences, including punctuation.  (I ask people to do this for all code comments, but it is especially important in API definitions because newlines get omitted in the `oc explain` output.)  The fact that syslog is used for container logging is an implementation detail, so it doesn't need to be mentioned in the API godoc.  
```suggestion
	// maxLength is the maximum length of log message.
```